action.yml

```
@@ -10,40 +10,63 @@ inputs:
     description: 'List of tags that gets passed along to the docker build args'
     required: true
     default: '' # set to an empty string
-  docker_username:
-    description: 'Docker username obtained from secrets'
+  registry_name:
+    description: 'Name of the registry action is loging in to'
     required: true
-  docker_password:
-    description: 'Docker password obtained from secrets'
+  username:
+    description: 'Username of the registry obtained from secrets'
+    required: true
```

```suggestion
  registry_user:
    description: 'Username of the registry obtained from secrets'
    required: true
```

This would clarify what `username` the variable is for